vuejs/src/App.vue

```diff
@@ -0,0 +1,9 @@
+<script setup>
```

Comment:
Can this entire file be removed and the TextEditor be inlined? I try to keep the demos as simple as possible.

Reply:
done